x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/categorization_field/categorization_field_select.tsx

```diff
@@ -17,12 +22,14 @@ interface Props {
 }
 
 export const CategorizationFieldSelect: FC<Props> = ({ fields, changeHandler, selectedField }) => {
-  const options: EuiComboBoxOptionProps[] = fields
-    .filter(f => f.id !== EVENT_RATE_FIELD_ID && f.type === ES_FIELD_TYPES.KEYWORD)
-    .map(f => ({
-      label: f.name,
-    }))
-    .sort((a, b) => a.label.localeCompare(b.label));
+  const { jobCreator } = useContext(JobCreatorContext);
+  const options: EuiComboBoxOptionProps[] = [
```

Comment:
Is there an issue when there is only one `keyword` type field? For example, farequote has 1 keyword field, but the dropdown is empty:

![image](https://user-images.githubusercontent.com/7405507/67399501-40ac8300-f5a4-11e9-9c7d-0b59a8d528ff.png)

And shouldn't it be possible to select `text` types too as the categorization field?

Reply:
This is a combination of the EUI combo box single item selection and missing text types which will be added in follow PRs